run.sh

```diff
@@ -10,6 +10,8 @@ sudo rm -rf /usr/local/apisix/
 rm -rf /tmp/apisix
 
 wget https://github.com/apache/apisix/raw/master/rockspec/apisix-$version-$iteration.rockspec -O apisix-$version-$iteration.rockspec
+sudo luarocks config variables.OPENSSL_LIBDIR /usr/local/openresty/openssl/lib/
+sudo luarocks config variables.OPENSSL_INCDIR /usr/local/openresty/openssl/include
```

Comment:
@Fuchange just for prettier code:

```suggestion
sudo luarocks config variables.OPENSSL_INCDIR /usr/local/openresty/openssl/include/
```

Reply:
:ok_hand: done